data-governance/no-code-access.mdx

```diff
@@ -3,19 +3,64 @@ title: "No code access needed"
 icon: "code"
 ---
 
-We operate by simulating a real user interacting with your web app, using your app's DOM to identify actions like clicks or text inputs. This doesn't require access to your code.
+## No code access
 
-Our [GitHub Action](/quickstart-GitHub) or [Azure DevOps Task](/quickstart-Azure) initiates your test suite, requiring an `API key`, `test target ID` and `target URL`.
+We operate by simulating a real user interacting with your web app, using your app's DOM to identify actions like clicks or text
+inputs. We are also using screenshots. This doesn't require access to your code base. We are only seeing those parts of your app
+which are exposed to your users anyways.
+Hence we are not jepordizing any SOC2 or ISO compliance requirements you might need to follow.
 
-- The **API key** identfies your organization
-- The **test target ID** identfies your test suite
-- Tests are performed on the **target URL**
+## GDPR compliance
+
+We are not GDPR relevant since we are not collecting any user data (privacy relevant or not). You can strictly run
+Octomind on test data to make sure, your are not violating your own compliance requirements.
+
+## One IP address
+
+If you need to set your firewall to allow our agent to exercise your web app, please use this IP address: `35.192.162.70`
+
+<Warning>
+  Please let us know so that we can set your organisation to always use our
+  proxy. We do not have this option as self-service yet.
+</Warning>
+
+## Metrics
+
+In case you need to exclude our agent from your metrics you can easily do this by excluding any browsers with a browser name
+extension of `octomind`.
+
+## CI/CD Integration
```

Comment:
why are we explaining this here if it's on a different page already? can we make it a snippet so its reusable if we need it in both spots?

Reply:
yes, good idea. We already have it? I thought it was new. anyways. I will resolve this conversation since it is the only thing I can technically do.

Reply:
yes? https://octomind.dev/docs/integrations-overview + the specific integration page? I'm just saying if you want to have it here too, please create it as a snippet so at least we only need to change it once

Reply:
ok, reduced to the bare minimum in terms of data gonvernance